instrument_config/mwa_beam_gaussian_decomp.pro

```diff
@@ -0,0 +1,136 @@
+    p = [$
+      1.00,cen,pix_hor/13.3,cen,pix_hor/13.2,$ ;pr
+      -0.07,cen,pix_hor/14.4,cen-pix_hor/6.45,pix_hor/22.1,$ ;pr s null
+      -0.07,cen,pix_hor/14.4,cen+pix_hor/6.47,pix_hor/22.1,$ ;pr n null
+      -0.1,cen-pix_hor/7.4,pix_hor/18,cen,pix_hor/13.8,$ ;pr w null
+      -0.1,cen+pix_hor/6.2,pix_hor/18,cen,pix_hor/13.8,$ ;pr e null
+      0.066,cen-pix_hor/60.8,pix_hor/15.2,cen+pix_hor/3.7,pix_hor/25,$ ;sn1
+      0.021,cen-pix_hor/29.3,pix_hor/16.2,cen+pix_hor/2.3,pix_hor/38.5,$ ;sn2
+      0.066,cen+pix_hor/63.5,pix_hor/15.2,cen-pix_hor/3.7,pix_hor/24.9,$ ;ss1
+      0.021,cen+pix_hor/61.6,pix_hor/16.5,cen-pix_hor/2.3,pix_hor/38,$ ;ss2
+      0.067,cen-pix_hor/3.89,pix_hor/22.5,cen-pix_hor/64,pix_hor/14.4,$ ;sw1
+      0.017,cen-pix_hor/2.2,pix_hor/29.1,cen-pix_hor/38,pix_hor/14.8,$ ;sw2
+      0.067,cen+pix_hor/4.5,pix_hor/17.4,cen+pix_hor/72,pix_hor/14.1,$ ;se1
+      0.001,cen+pix_hor/2.37,pix_hor/93.4,cen+pix_hor/46,pix_hor/18.5,$;se2
+      0.002,cen+pix_hor/4.0,pix_hor/34.75,cen+pix_hor/3.5,pix_hor/32.9,$ ;sne1
+      0.004,cen-pix_hor/3.55,pix_hor/30.3,cen+pix_hor/3.9,pix_hor/30.6,$ ;snw1
+      0.002,cen+pix_hor/3.57,pix_hor/34.6,cen-pix_hor/3.9,pix_hor/32.7,$ ;sse1
+      0.004,cen-pix_hor/4.0,pix_hor/30.4,cen-pix_hor/3.4,pix_hor/30.6];ssw1
+  endif
+  if (pointing EQ -2) OR (pointing EQ 2) then begin
+    ;; Params are for pointing -2. However, this can also be the params for 
+    ;;  pointing 2 if they are flipped about the x-axis below
+    p = [$
+      1.00,cen,pix_hor/12.6,cen,pix_hor/12.6,$ ;pr
+      -0.1,cen,pix_hor/13.9,cen+pix_hor/6.8,pix_hor/16,$ ;pr s null
+      -0.1,cen,pix_hor/13.9,cen-pix_hor/6.72,pix_hor/16,$ ;pr n null
+      -0.14,cen+pix_hor/6.3,pix_hor/15.5,cen,pix_hor/13.4,$ ;pr w null
+      -0.14,cen-pix_hor/7.3,pix_hor/15.5,cen,pix_hor/13.4,$ ;pr e null
+      0.078,cen+pix_hor/37,pix_hor/14.8,cen-pix_hor/3.87,pix_hor/22,$ ;ss1 part2
+      0.018,cen+pix_hor/35,pix_hor/17.9,cen-pix_hor/2.3,pix_hor/39.7,$ ;ss2
+      0.078,cen-pix_hor/30,pix_hor/14.7,cen+pix_hor/3.88,pix_hor/22.3,$ ;sn1 part2
+      0.018,cen-pix_hor/13.8,pix_hor/16.6,cen+pix_hor/2.3,pix_hor/41.6,$ ;sn2
+      ;0.007, cen+pix_hor/3.1,pix_hor/5170,cen-pix_hor/19.5,  pix_hor/21.8,$ ;se1 part2
+      0.09,  cen+pix_hor/4.8, pix_hor/16.9,cen+pix_hor/45,  pix_hor/13.6,$ ;se1 part2
+      ;se2
+      0.086, cen-pix_hor/3.89,pix_hor/20.6,cen-pix_hor/33,  pix_hor/14.2,$ ;sw1 part2
+      0.031, cen-pix_hor/2.14,pix_hor/26.9,cen-pix_hor/18.1,pix_hor/14.9,$ ;sw2
+      0.005, cen-pix_hor/4.2, pix_hor/29.3,cen-pix_hor/3.26,pix_hor/30.4,$ ;sse1
+      0.005, cen-pix_hor/3.3, pix_hor/29.3,cen+pix_hor/4.13,pix_hor/30.6] ;snw1
+  endif
+  
+  ;Model was made at high-band center frequency, scale appropriately
+  freq_dep = 1.82435e8 / freq
+  var = reform(p,5,N_elements(p)/5.)
+  var[1,*] = ((var[1,*] - cen) * freq_dep) + cen
+  var[3,*] = ((var[3,*] - cen) * freq_dep) + cen
+  var[2,*] *= freq_dep
+  var[4,*] *= freq_dep
```

Comment:
As in other functions, please break `var` apart into 5 appropriately-named variables.